### DISCUSSION TITLE
Integration into project management tools

### DISCUSSION THREAD

**Discussion by jonasschultheiss** [solidtime-io/solidtime#272] [Feature requests]
I'm a part time student interested in being self employed. At this stage and with limited time I don't want to invest a lot of money into things. This is why I really like OSS that is self hostable, like yours.
I was wondering, if you evalated integrations into popular project management tools? I really like Linear and their app. If were to implement an integration, users could import projects/tasks from linear and track the time directly in the project management tool

**Comment by mmoollllee** [comment]
Hey solidtime team!
I'm very interested in using solidtime too! What @jonasschultheiss mentioned is one thing that's still stopping me from doing so. Any plans of implementing external APIs to fetch Projects and Tasks from? Would be nice if the Founders could sketch a skeleton, so the community can build and share own integrations ;)

**Comment by moromidesign** [comment]
Our team is currently switching to solidtime from Toggle as a live trial. A linear integration would be a bonus for us. +1 from here.

**Reply by moromidesign** [reply]
We are interested in an integration that supports two-way creation and editing of tasks so that they are created in linear but also show up in solidtime.
